common/djangoapps/third_party_auth/tests/test_views.py

```diff
@@ -81,6 +81,7 @@ def test_signed_metadata(self, key_name, pub_key_starts_with):
         self._validate_signed_metadata(pub_key_starts_with=pub_key_starts_with)
 
     def test_secure_key_configuration(self):
+        pass
```

Comment:
I don't think you need this here :)

Reply:
I don't think so either 😄  I wonder how did this slip throw!